integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/BridgeSteps.java

```diff
@@ -111,6 +112,24 @@ public void createFakeBridge(String testBridgeName) {
         bridgeContext.setDeleted(true);
     }
 
+    @When("^update the Bridge \"([^\"]*)\" with body:$")
+    public void updateBridge(String testBridgeName, String bridgeRequestJson) {
+        BridgeContext bridgeContext = context.getBridge(testBridgeName);
+        String systemBridgeName = BridgeResource.getBridgeDetails(context.getManagerToken(), bridgeContext.getId()).getName();
```

Comment:
AFAIK you can get Bridge system name using `bridgeContext.getName()`. Can you try it?

Reply:
Yes it worked , Thanks